dotcom-rendering/src/lib/themeToPillar.ts

```diff
@@ -0,0 +1,22 @@
+import { ArticlePillar, ArticleSpecial } from '@guardian/libs';
+
+export const themeToPillar = (
+	theme?: ArticleTheme,
+): ArticlePillar | undefined => {
+	switch (theme) {
+		case ArticleSpecial.SpecialReport:
+		case ArticleSpecial.SpecialReportAlt:
```

Comment:
Should these also be `undefined`?

Reply:
I thought these tend to be "News"? e.g. we wouldn't special report about a restaurant review would we? 
But you're right we should be consistent with what Frontend was doing, rather than go with my vague impressions about what should be classed as news :D

Reply:
I don't think they're always guaranteed to be news. For example, here are a special report and special report alt in the opinion pillar:

- https://www.theguardian.com/commentisfree/2016/apr/10/the-guardian-view-on-david-cameron-the-warping-ways-of-wealth
- https://www.theguardian.com/commentisfree/2023/mar/28/the-guardian-view-on-our-links-to-slavery-a-necessary-reckoning-with-the-past